pkg/operator/ceph/operator.go

```diff
@@ -117,6 +118,10 @@ func (o *Operator) Run() error {
 		}
 	}
 
+	err := admission.InitWebhookIfSecretPresent(o.context, o.rookImage)
+	if err != nil {
+		logger.Errorf("failed to start webhook. %v", err)
```

Comment:
If the webhook is configured to start, let's fail the operator from starting. 
```suggestion
		return errors.Wrap(err, "failed to start webhook")
```